slam-ws/src/nav_rtab/CMakeLists.txt

```diff
@@ -0,0 +1,202 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
We've been removing all the unnecessary comments from these files in our other packages, can you do the same here